include/picongpu/algorithms/DifferenceToLower.def

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2013-2018 Heiko Burau, Rene Widera, Axel Huebl
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+
+#include <pmacc/math/Vector.hpp>
+
+
+namespace picongpu
+{
+
+    /** calculate difference to lower value
+     *
+     * @tparam T_Dim for how many dimensions this operator access memory
```

Comment:
REQUIRED: 
The wording is unclear 
do you mean
number of dimensions the operator accesses in memory
?